superset-frontend/src/dashboard/components/DashboardBuilder/DashboardContainer.tsx

```diff
@@ -72,7 +72,7 @@ const DashboardContainer: FC<DashboardContainerProps> = ({ topLevelTabs }) => {
             activeKey={activeKey}
             renderTabBar={() => <></>}
             fullWidth={false}
-            animated
+            animated={false}
```

Comment:
I disabled animation as it was breaking native filters - when user switched tab, `margin-left: 100%` style was set on tab content, which was covering native filters panel, causing the user to be unable to click native filters... And the animation looked ugly with native filters panel anyway 😛